docs/api/sql/Function.md

```diff
@@ -1105,4 +1105,42 @@ Result:
 +---------------------------------------------------------------+
 |LINESTRING (3 6, 2 4, 1 2, 0 0)                                |
 +---------------------------------------------------------------+
-```
\ No newline at end of file
+```
+
+## ST_XMax
+
+Introduction: Returns the maximum X coordinate of a geometry
+
+Format: `ST_XMax (A:geometry)`
+
+Since: `v1.2.1`
+
+Example:
+
+```SQL
+SELECT ST_XMax(df.geometry) AS xmax
+FROM df
+```
+
+Input: `POLYGON ((-1 1, 1 -1, -1 -1, 1 1))`
```

Comment:
Polygon example here is not a WKT format. Use POLYGON ((-1 -11, 0 10, 1 11, 2 12, -1 -11)) instead.

Please fix other places in the docs

Reply:
Changed example in all docs